AdyenEncryption/Extensions/DataExtension.swift

```diff
@@ -13,5 +13,16 @@ extension Data {
         let count = self.count / MemoryLayout<CUnsignedChar>.size
         return [CUnsignedChar](UnsafeBufferPointer<CUnsignedChar>(start: start, count: count))
     }
-
+    
+    internal init?(base64URLEncoded base64URLString: String) {
+        self.init(base64Encoded: base64URLString.base64String())
+    }
+    
+    internal func base64URLString() -> String {
+        var result = base64EncodedString()
+        result = result.replacingOccurrences(of: "+", with: "-")
```

Comment:
can we use here `base64String()` from String?

Reply:
this function transforms a base64 to base64Url, but `String.base64String()` dose the opposite, it transforms base64Url into base64